packages/client-engine-runtime/src/transactionManager/TransactionManager.ts

```diff
@@ -104,13 +105,17 @@ export class TransactionManager {
     // Transaction status might have changed to timed_out while waiting for transaction to start. => Check for it!
```

Comment:
```suggestion
    // Transaction status might have timed out while waiting for transaction to start. => Check for it!
```
since we're not actually doing that by checking for the state transition to `timed_out` in `case 'waiting'` anymore